app/cinema/hall.py

```diff
@@ -0,0 +1,14 @@
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self, movie_name: str,
+                      customers: str, cleaning_staff: str) -> None:
```

Comment:
'customers' should be list Customer type and cleaning staff should be Cleaner type
```suggestion
    def movie_session(self, movie_name: str,
                      customers: list[Customer], cleaning_staff: Cleaner) -> None:
```